storm-core/src/ui/public/index.html

```diff
@@ -143,6 +143,12 @@
             ]
           });
           $('#topology-summary [data-toggle="tooltip"]').tooltip();
+          $.getJSON("/api/v1/cluster/configuration", function(json){
+              var displayResource = json["scheduler.display.resource"];
+              if (!displayResource){
+                  $('#topology-summary td:nth-child(10),#topology-summary th:nth-child(10)').hide();
```

Comment:
We could modify the template to decide not list headers and values id `displayResource` is `false`. That is cleaner that populating and hiding certain columns by number.